modules/widgets/flickr/form.php

```diff
@@ -0,0 +1,81 @@
+<p>
+	<label>
+		<?php esc_html_e( 'Title:', 'jetpack' ); ?>
+	</label>
+	<input
+		class="widefat"
+		name="<?php echo esc_attr( $this->get_field_name( 'title' ) ); ?>"
+		type="text"
+		value="<?php echo esc_attr( $instance['title'] ); ?>"
+	/>
+</p>
+
+<p>
+	<label>
+		<?php esc_html_e( 'Flickr RSS URL:', 'jetpack' ); ?>*
+	</label>
+	<input
+		class="widefat"
+		name="<?php echo esc_attr( $this->get_field_name( 'flickr_rss_url' ) ); ?>"
+		type="text"
+		value="<?php echo esc_attr( $instance['flickr_rss_url'] ); ?>"
+	/>
+</p>
+
+<p>
+	<label>
+		<?php esc_html_e( 'How many photos would you like to display?', 'jetpack' ); ?>
+	</label>
+	<select name="<?php echo esc_attr( $this->get_field_name( 'items' ) ); ?>">
+		<?php for ( $i = 1; $i <= 10; ++$i ) { ?>
+			<option
+				<?php selected( $instance['items'], $i ); ?>
+				value="<?php echo $i; ?>"
+			>
+				<?php echo $i; ?>
+			</option>
+		<?php } ?>
+	</select>
+</p>
+
+<p>
+	<label>
+		<?php esc_html_e( 'What size photos would you like to display?', 'jetpack' ); ?>
+	</label>
+	<select name="<?php echo esc_attr( $this->get_field_name( 'flickr_image_size' ) ); ?>">
+		<?php
+		$flickr_sizes = array(
+			array(
+				'size' => 'thumbnail',
+				'text' => esc_html__( 'Thumbnail', 'jetpack' ),
+			),
+			array(
+				'size' => 'small',
+				'text' => esc_html__( 'Small', 'jetpack' ),
+			),
+		);
+		foreach( $flickr_sizes as $flickr_size ) { ?>
+			<option
+				<?php selected( $instance['flickr_image_size'], $flickr_size['size'] ); ?>
+				value="<?php echo esc_attr( $flickr_size['size'] ); ?>"
+			>
+				<?php esc_html_e( $flickr_size['text'] ); ?>
+			</option>
+		<?php }
+		?>
+	</select>
+</p>
+
+<p>
+	<small>*
+		<?php esc_html_e( 'To find your Flickr RSS feed, go to your photostream and click the "Edit" tab at the top. Scroll down to the bottom of the page until you see the RSS icon or the Latest link. Right click on either and copy the URL. Paste into the box above.', 'jetpack' ); ?>
```

Comment:
<img width="969" alt="captura de pantalla 2017-04-07 a las 17 41 27" src="https://cloud.githubusercontent.com/assets/1041600/24819151/d4f00a54-1bb9-11e7-813f-16fb858cb04c.png">

a) This should be updated since there are no tabs now. There's a More link that unfolds a menu. Edit is now there.

b) Would be great if the text could be trimmed down to a point where it fits nicely below the input field, even in more than one line.